docs/release/release_0_4_17.md

```diff
@@ -42,6 +42,7 @@ https://github.com/napari/napari
 - add ci for asv benchmark suite (#4554)
 - Disconnect a removed slider's _pull_label callback from the axis label change event  (#4557)
 - Fixes/modifications to nested list model and Group in prep for layergroups (#4560)
+- [Automatic] Update albertosottile/darkdetect vendored module (#4561)
```

Comment:
I had to manually add this line. Based on the behavior described in #4562 I believe this was not included in v0.4.16, but I have not followed the tags/commits to understand why this was skipped by the release notes generation script. Maybe because [it tries to skips contribution from bots](https://github.com/napari/napari/blob/fe4dfb7b798a75f36df7182f66f74141744eb863/docs/release/generate_release_notes.py#L119)?